src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java

```diff
@@ -210,8 +211,10 @@ private static int startGame(CommandContext<ServerCommandSource> context) throws
                 Text error = startResult.getError();
                 message = error.shallowCopy().formatted(Formatting.RED);
             } else {
-                message = source.getDisplayName().shallowCopy().append(" has started the game!")
-                        .formatted(Formatting.GRAY);
+                message = new TranslatableLiteralText(source.getDisplayName().shallowCopy().append(" has started the game!"),
```

Comment:
Hmm- would it be possible to have the messages not duplicated?

Reply:
It's the default message used when Server-Translation is not on the server.

Reply:
It depends if you prefer Server-Translations to be a hard dependency or a soft one.

Reply:
Ah okay, I understand. Might it make sense to have the mod be JIJ'd? Or if not JIJ'd completely, a small part of the API that delegates to en_us translation if not installed?

Reply:
Opened #39 for the not JIJ'd completely case.